src/utils/electronStore.ts

```diff
@@ -216,6 +216,31 @@ const schema = {
       },
     },
   },
+  pluginConfig: {
```

Comment:
@devcatalin In follow up PR we should sort the whole plugin approach, but since this is focused on UI I just "made it work".

Reply:
sure, this is fine for now